package.json

```diff
@@ -81,13 +81,13 @@
     "peer-id": "^0.14.2",
     "reflect-metadata": "^0.1.13",
     "rlp": "^2.2.6",
-    "sequelize": "^5.21.10",
+    "rxjs": "^6.6.3",
```

Comment:
haha I think I suggested Observable  back in March/April

Reply:
Hmm don't recall, but it is here only for the tracking progress at https://www.npmjs.com/package/listr library